build.gradle

```diff
@@ -16,3 +16,73 @@ allprojects {
         jcenter()
     }
 }
+
+task replaceGatewayIFAPITestVersion {
```

Comment:
better to abstract the same logic for `replaceGatewayIFAPITestVersion`, `replaceThingIFAPITestVersion`, and `replaceThingIFAPIVersion`